ui/src/app/modules/boards/pages/login/login.page.html

```diff
@@ -21,20 +21,16 @@
         <div class="rq-team-controls">
             <form id="loginForm" class="rq-form">
                 <div class="rq-form-header">
-                    <div class="header-text">Sign in to your Board!</div>
-                    <a id="createBoard" routerLink="/create"
-                        >or create a new board</a
-                    >
+                    <div class="header-text">Sign in to your Team!</div>
+                    <a id="createBoard" routerLink="/create">or create a new team</a>
                 </div>
 
-                <form
-                    class="rq-form-body"
-                    (keydown.enter)="requestCaptchaStateAndLogIn()"
-                >
+                <form class="rq-form-body" (keydown.enter)="requestCaptchaStateAndLogIn()">
                     <div class="rq-input-group">
-                        <label for="teamNameInput">Board name</label>
+                        <label for="teamNameInput">Team name</label>
                         <input
                             id="teamNameInput"
+                            data-testid="teamNameInput"
```

Comment:
Is there a reason we're putting test IDs on these instead of using the IDs already on the element?

Reply:
For cypress tests. It's best practice and way easier to maintain for E2E tests to not rely on ids, tags, or classes. In the long term it saves so much headache. Trust me.

https://docs.cypress.io/guides/references/best-practices#Selecting-Elements

Reply:
For this then maybe we should do it on placeholder or label instead?

Reply:
Alright, looks good then.